synapse/config/oidc_config.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright 2020 Quentin Gliech
+# Copyright 2020 The Matrix.org Foundation C.I.C.
```

Comment:
```suggestion
# Copyright 2020-2021 The Matrix.org Foundation C.I.C.
```

Reply:
this shows how long it's been sitting in a branch in git... I've actually fixed it in later commits.